### ISSUE TITLE
The plugin ignores to check folders where only index.{js,ts} is present

### ISSUE THREAD

**Issue by Valdermeyder** [DukeLuo/eslint-plugin-check-file#10]
# Steps to reproduce

- Follow the installation process
- Add rule

```
   "check-file/folder-naming-convention": [
      "error",
      {
        "*/!(__tests__)/**/": "KEBAB_CASE"
      }
    ]
```
- Create a folder with "NotKebabCase" and create "index.js" inside it
- Run a linter

# Current behavior
No error is reported

# Expected behavior
"NotKebabCase" folder should be reported as eslint error

**Comment by DukeLuo** [comment]
I think you are using the wrong glob expression. If you want to match all the folders except `__tests__`, please use the glob expression `!(__tests__)/**/`. Worked for me.

**Comment by Valdermeyder** [comment]
The expression you are proposing doesn't really work. When I use it I'm getting the error below for every file

> 1:1  error  The folder "." does not match the "KEBAB_CASE" style  check-file/folder-naming-convention

Maybe it is because of the folder where my project is cloned which is not in kebab-case, but all directories inside are.

**Comment by Valdermeyder** [comment]
It seems like the solution for the issue is to set `"root": true` inside the top-level .eslint configuration file.

However, another problem appears when `eslint .` (run for the current directory) is used
```
The folder "." does not match the "KEBAB_CASE" style
```

**Comment by DukeLuo** [comment]
@Valdermeyder As I do not know much about `lerna`, can you help to set up a reproducible repository for this issue?
